stacks/list.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/Scalingo/cli/config"
 )
 
-func List(ctx context.Context) error {
+func List(ctx context.Context, withDeprecated bool) error {
```

Comment:
Nitpick: I don't find the name `withDeprecated` very representative for this variable.
Maybe something like `isWithDepractedFlag` better, WDYT?

Reply:
I'll do the change